Adding all new demos to release/3.0 (#777)
Avaiga/taipy-doc: docs/knowledge_base/demos/background_remover.md

Background Remover is an image processing application built using Taipy. The objective is
to simplify the process of removing backgrounds from images. This demo showcases how you
can achieve this effortlessly using Taipy, and it offers a peek into the code.

[Try it live](https://background-remover.taipy.cloud/){: .tp-btn target='blank' }
[Get it on GitHub](https://github.com/Avaiga/demo-remove-background){: .tp-btn .tp-btn--accent target='blank' }

![Background Remover](images/background-remover.png){width=80%}

# Understanding the Application

**Application Overview:**
This one-page demo illustrates how you can create a background removal tool with minimal
effort using Taipy. The application is designed to perform background removal on uploaded
images, providing a quick and clean result.

**Key Features:**

- **Background Removal:** Automatically detects and removes the background from uploaded
photos.
- **Interactive User Interface:** Enables users to upload images and instantly see the
background-removed output.
- **Code Efficiency:** Achieves the background removal task in just a few lines of Python
code.

> This one page demo uses rembg and [OpenCV library](https://opencv.org/) to remove the

Reviewer: ```suggestion
This one page demo uses `rembg` and [the OpenCV library](https://opencv.org/) to remove the
```
A link to the rembg repo?

background of pictures.

# How to Use the Application

Using Background Remover is straightforward:

> 1. **Upload Your Image:** Select an image that you want to remove the background; this

Reviewer: ```suggestion
1. **Upload Your Image:** Select an image where you want to remove the background; this
```

Reviewer 2: "In which" instead of where or that.

will start the processing.

> 2. **More options:** Some parameters influences how the background is removed. These

Reviewer: ```suggestion
2. **More options:** Some parameters influence how the background is removed. These
```

> parameters can be changed in the "More options" expandable.

Reviewer: ```suggestion
parameters can be changed in the "More options" expandable section.
```

3. **Instant Result:** The processed image with the background removed will be displayed
below.

> 4. **Download:** You can download the image through a button.

Reviewer: ```suggestion
4. **Download:** You can download the image by pressing a button.
```
